apps/nextjs-website/src/__tests__/helpers/EmailFormWrapper.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { NextIntlClientProvider } from 'next-intl';
+import { PropsWithChildren } from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import EmailFormWrapper from '@/components/organisms/EmailFormWrapper/EmailFormWrapper';
+import messages from '@/messages/it.json';
+
+const Wrapper = ({ children }: PropsWithChildren) => (
+  <NextIntlClientProvider locale={'it'} messages={messages}>
+    {children}
+  </NextIntlClientProvider>
+);
```

Comment:
I think this wrapper will be used to every component's test ? In that case i suggest to move it to a separate file